src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/ForcibleFinishExceptionInTryBlockTest.kt

```diff
@@ -8,15 +8,16 @@
  * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-package org.jetbrains.kotlinx.lincheck.test.representation
+package org.jetbrains.kotlinx.lincheck_test.representation
 
+import junit.framework.Assert.assertTrue
 import kotlinx.atomicfu.*
 import org.jetbrains.kotlinx.lincheck.*
 import org.jetbrains.kotlinx.lincheck.annotations.*
 import org.jetbrains.kotlinx.lincheck.strategy.*
 import org.jetbrains.kotlinx.lincheck.strategy.managed.*
 import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.*
-import org.jetbrains.kotlinx.lincheck.test.*
+import org.jetbrains.kotlinx.lincheck_test.*
 import org.junit.*
 
 class ForcibleFinishExceptionInTryBlockTest {
```

Comment:
What if we add an empty `catch(t: Throwable) {}` block?

Reply:
We will fail with proper result anyway, but a little bit later.